e2e/interchaintestv8/e2esuite/suite.go

```diff
@@ -68,17 +105,22 @@ func (s *TestSuite) SetupSuite(ctx context.Context) {
 	}))
 
 	// map all query request types to their gRPC method paths for cosmos chains
-	s.Require().NoError(populateQueryReqToPath(ctx, s.ChainB))
+	s.Require().NoError(PopulateQueryReqToPath(ctx, s.ChainB))
+
+	if s.ethTestnetType == TestnetTypePoW {
+		anvil := chains[1].(*icethereum.EthereumChain)
+		faucet, err := crypto.ToECDSA(ethcommon.FromHex("0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"))
+		s.Require().NoError(err)
+
+		s.ChainA, err = ethereum.NewEthereum(ctx, anvil.GetHostRPCAddress(), nil, faucet)
+		s.Require().NoError(err)
+	}
 
 	// Fund user accounts
 	cosmosUserFunds := sdkmath.NewInt(testvalues.InitialBalance)
 	cosmosUsers := interchaintest.GetAndFundTestUsers(t, ctx, t.Name(), cosmosUserFunds, s.ChainB)
 	s.UserB = cosmosUsers[0]
-	ethUsers := interchaintest.GetAndFundTestUsers(t, ctx, t.Name(), testvalues.StartingEthBalance, s.ChainA)
-	s.UserA = ethUsers[0]
 
-	t.Cleanup(
-		func() {
-		},
-	)
```

Comment:
Is there nothing to add to cleanup here?

Reply:
No, I don't think so